icons/radar.svg

```diff
@@ -0,0 +1,19 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m13.4 10.6 5.7-5.7C17.3 3.1 14.8 2 12 2c-1.8 0-3.5.5-5 1.4" />
+  <path d="M2.3 9.6c-.2.8-.3 1.6-.3 2.4 0 5.5 4.5 10 10 10s10-4.5 10-10c0-1.3-.3-2.5-.7-3.6" />
+  <path d="M4 6h.01" />
+  <path d="M16.2 7.8c-2.3-2.3-6.1-2.3-8.5 0-2.3 2.3-2.3 6.1 0 8.5.2.2.3.3.5.4" />
+  <path d="M15.8 16.7c1.4-1.1 2.2-2.8 2.2-4.7" />
+  <path d="M12 18h.01" />
+  <circle cx="12" cy="12" r="2" />
```

Comment:
It took me to completely redraw this from scratch and do some heavy manual optimization but I finally got it down to something quite small that's still on the grid:
```suggestion
  <path d="M19.08 4.92A9.97 9.97 0 0 0 6.98 3.35" />
  <path d="M4 6h.01" />
  <path d="M21.31 8.35A10 10 0 1 1 2.28 9.62" />
  <path d="M16.24 7.76A6 6 0 1 0 8.23 16.67" />
  <path d="M12 18h.01" />
  <path d="M17.99 11.66A6 6 0 0 1 15.77 16.67" />
  <circle cx="12" cy="12" r="2" />
  <path d="m13.41 10.59 5.66-5.66" />
```

Reply:
🤌

Reply:
> It took me to completely redraw this from scratch and do some heavy manual optimization but I finally got it down to something quite small that's still on the grid:

@karsa-mistmere Nice!